frontend/src/app/planner/components/SelectProjectModal.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+import { Autocomplete, Sheet, Typography, Modal } from '@mui/joy'
+
+import { Button } from '@mui/joy'
+import { Project } from '@/domain/Project'
+
+type SelectProjectProps = {
+  open: boolean
+  projects: Array<Project>
+}
+
+export const SelectProjectModal = ({ open, projects }: SelectProjectProps) => {
+  const router = useRouter()
+  const [selectedProject, setSelectedProject] = useState<Project | null>(null)
+
+  return (
+    <Modal
+      sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
+      open={open}
+      aria-labelledby="confirmation-modal-title"
+      aria-describedby="modal-content"
+    >
+      <Sheet
+        sx={{
+          minWidth: 300,
+          minHeight: 200,
+          borderRadius: 'md',
+          padding: 3,
+          boxShadow: 'lg',
+          display: 'flex',
+          flexDirection: 'column',
+          gap: '8px'
+        }}
+      >
+        <Typography
+          component="h2"
+          id="confirmation-modal-title"
+          level="h4"
+          textColor="inherit"
+          fontWeight="lg"
+        >
+          Select Project To Manage
+        </Typography>
+        <Autocomplete
```

Comment:
Since the autocomplete is a form field, it needs a label. You can add an `aria-labelled-by="confirmation-modal-title"` or a form label. Material recommends a form label: https://mui.com/joy-ui/react-autocomplete/#accessibility. I also prefer form labels as they are the best for a11y. So, perhaps the best way to go is to change the <Typography> to a <FormLabel> and add the `confirmation-modal-title`id to that so that the modal label can pull from it. If you use <FormLabel>, you will also need to wrap it all in a <FormControl> as shown here: https://mui.com/joy-ui/react-autocomplete/#label

You can add extra styles to the label if you want to bold it and make it a bit larger.

Reply:
Done, thanks for catching this